README.md

```diff
@@ -61,20 +62,29 @@ circ.measure_all()
 aersim = AerSimulator()
 
 # Perform an ideal simulation
-result_ideal = qiskit.execute(circ, aersim).result()
+result_ideal = aersim.run(circ, shots=1000).result()
 counts_ideal = result_ideal.get_counts(0)
 print('Counts(ideal):', counts_ideal)
 # Counts(ideal): {'000': 493, '111': 531}
 
 # Construct a noisy simulator backend from an IBMQ backend
 # This simulator backend will be automatically configured
 # using the device configuration and noise model 
-provider = IBMQ.load_account()
-backend = provider.get_backend('ibmq_athens')
-aersim_backend = AerSimulator.from_backend(backend)
+provider = IBMProvider()
+hub = "ibm-q"
+group = "open"
+project = "main"
+backend_name = "ibmq_lima"
+backend = provider.get_backend(backend_name, instance=f"{hub}/{group}/{project}")
```

Comment:
I think README example should be simple. Could you change the codes to get `aersim_backend` as follows? 
```
provider = IBMProvider(token='FILL_WITH_YOUR_TOKEN')
backend = provider.get_backend('ibmq_lima')
aersim_backend = AerSimulator.from_backend(backend)
```

Reply:
Thanks. I made the change as you suggested and also added from qiskit.providers.fake_provider import FakeVigo. The code runs well in IBM-q well.